driver/conv_driver.hpp

```diff
@@ -1048,6 +1106,14 @@ int ConvDriver<Tgpu, Tref>::SetConvDescriptorFromCmdLineArgs()
         convDesc, spatial_dim, pads.data(), conv_strides.data(), conv_dilations.data(), mode);
 
     miopenSetConvolutionGroupCount(convDesc, group_count);
+    if(inflags.GetValueStr("compute_type") != "-1")
+    {
+        miopenDataType_t compute_type =
+            DataTypeFromShortString(inflags.GetValueStr("compute_type"));
+        // miopenSetConvolutionComputeType(convDesc, compute_type);
+        miopenSetConvolutionAttribute(
+            convDesc, MIOPEN_CONVOLUTION_ATTRIB_COMPUTE_TYPE, static_cast<int>(compute_type));
```

Comment:
This is not implemented. I highly recommend moving this incomplete functionality to another PR.

Reply:
Fixed!